### PR TITLE
feat: Small tweak around getting the bucket number

### PR DESCRIPTION
**Pull request by polok**

### CODE REVIEW

File: Tests/FeaturevisorSDKTests/BucketTests.swift [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import Foundation
+import XCTest
+
+@testable import FeaturevisorSDK
+
+final class BucketTests: XCTestCase {
```
Comment (fahad19) [inline Tests/FeaturevisorSDKTests/BucketTests.swift:6]: at some point in future, we need to run the same assertions as found in the JS SDK here for the key/value pairs: https://github.com/fahad19/featurevisor/blob/249738ba6c44208a7174263ce2cbc26d23b88b3d/packages/sdk/src/bucket.spec.ts#L22-L29

if they pass, we will be sure that the bucketing is consistent across different language SDKs.

**Review by fahad19 — APPROVED** [review]

File: Sources/FeaturevisorSDK/Bucket.swift [hunk 2]
```diff
@@ -1,13 +1,18 @@
...
 import MurmurHash_Swift
 
-let HASH_SEED: UInt32 = 1
-let MAX_HASH_VALUE: Double = pow(2, 32)  // 2^32
-let MAX_BUCKETED_NUMBER = 100000  // 100% * 1000 to include three decimal places in the same integer value
+internal enum Bucket {
+    
+    private static let HASH_SEED: UInt32 = 1
+    private static let MAX_HASH_VALUE: Double = pow(2, 32)  // 2^32
+    private static let MAX_BUCKETED_NUMBER = 100000  // 100% * 1000 to include three decimal places in the same integer value
```
Comment (fahad19) [inline Sources/FeaturevisorSDK/Bucket.swift:9]: 💯

**Review by maxgallo — APPROVED** [review]
LGTM

File: Tests/FeaturevisorSDKTests/BucketTests.swift (hunk 1 above)
Comment (polok) [reply Tests/FeaturevisorSDKTests/BucketTests.swift:6]: Great, if we already have such test data for JS SDK, I will update the PR and use the same as at the end this was my goal to have the same test data to be sure that all our SDKs behave in the same way here. Also, was thinking to have the same test data to validate SDK behavior - something that we can discuss later

Comment (fahad19) [reply Tests/FeaturevisorSDKTests/BucketTests.swift:6]: yep, so the link above will help regarding consistent bucketing.

as for the full SDK behaviour, that's where the CLI will come in handy. in JS side, there's a test runner: https://featurevisor.com/docs/testing/#running-tests

The `example-1` project is what I use for testing SDK behaviour like a regular user: https://github.com/fahad19/featurevisor/tree/main/examples/example-1

From Swift side, we can also ship a `featurevisor-swift` CLI that can run the test specs in YAML. that way both JS and Swift (and others in future) can run their tests from a common set of specs.

Mentioned in README: https://github.com/featurevisor/featurevisor-swift#test-runner

Comment (polok) [reply Tests/FeaturevisorSDKTests/BucketTests.swift:6]: I updated the test data. Results are the same what is good

Comment (fahad19) [reply Tests/FeaturevisorSDKTests/BucketTests.swift:6]: excellent news! thanks @polok for this.

I was a bit worried about cross-platform consistent bucketing tbh 😅

**Pull request merged**